src/Core/Tools/Models/Business/ReferenceEvent.cs

```diff
@@ -243,4 +243,12 @@ public ReferenceEvent(ReferenceEventType type, IReferenceable source, ICurrentCo
     /// the value should be <see langword="null" />.
     /// </value>
     public string SignupInitiationPath { get; set; }
+
+    /// <summary>
+    /// The will hold the details around the plan cusomer is and the plan they are migrating to. For example, "Teams Starter → Teams, Enterprise".
+    /// </summary>
+    /// <value>
+    /// <see langword="null"/> when the event was not originated by an application.
```

Comment:
I think the following is accurate, unless the event isn't filed at all when a downgrade occurs.

```suggestion
    /// <see langword="null"/> when the event was not originated by an application,
    /// or when a downgrade occurred.
```

Reply:
added recommended comment